coopertunes/hparams/MelSpecVAE.py

```diff
@@ -1,7 +1,7 @@
 from pathlib import Path
 from typing import Any, Optional, Union
 
-from .hparams import HParams
+from coopertunes.hparams import HParams
```

Comment:
Okay, why we don't like relative imports?

Reply:
Without using specific IDE, not using relative imports could be more readable to python interpreter. Specific imports are more robust. For example, working with project by setting PYTHONPATH to project root, could provoke errors sometimes.